assets/php/class-wgpb-block-library.php

```diff
@@ -396,10 +396,20 @@ public static function print_script_settings() {
 			'default_height'    => wc_get_theme_support( 'featured_block::default_height', 500 ),
 			'isLargeCatalog'    => $product_counts->publish > 200,
 		);
+
+		// Checkout settings.
+		$packages = WC()->shipping->get_packages();
+
+		$checkout_settings = array(
+			'isUserShopManager' => current_user_can( 'manage_woocommerce' ),
+			'isCouponsEnabled'  => wc_coupons_enabled(),
```

Comment:
Nitpicking, but maybe `hasCouponsEnabled` for this var since it's plural.  And optionally `hasShippingEnabled` for below var.